### DISCUSSION TITLE
Add Pi-hole

### DISCUSSION THREAD

**Discussion by miguelgargallo** [coollabsio/coolify#1808] [Service Template Requests]
Literally, if you add pi-hole i remove my hole raspberry-pi at home and put coolify too

**Comment by cjnewbs** [comment]
I'm working on this. I have got it working and I'm just tweaking the configuration. Will probably PR in the next few days 😎

**Comment by cjnewbs** [comment]
https://github.com/coollabsio/coolify/pull/2628

**Reply by Rehaeel** [reply]
What left to be done for this beautiful initiative? 🙃
I'm patiently waiting

**Comment by honel** [comment]
I am struggling with the deployment of pihole on my coolify-server and would love to see pihole offered as a service.

**Comment by 2u841r** [comment]
@andrasbacsai please sir

**Comment by ddavid93** [comment]
@andrasbacsai some news about this discussion?

**Comment by mdibnzamir** [comment]
@andrasbacsai please add Pi-Hole

**Comment by djsisson** [comment]
https://docs.pi-hole.net/docker/

remove 80/443 from the ports section and add ```SERVICE_FQDN_PIHOLE``` to environment section
also remove anything from caps section you don't need
and change port 53 bind if you required

so just paste it in as a compose

**Comment by OG-Jons** [comment]
#6020 
Tried adding it, works on my side, but that's always relative...

**Reply by DanOpcode** [reply]
Not all heroes wear capes
